src/main/java/uk/gov/hmcts/reform/sandl/snlevents/controllers/HearingController.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.sandl.snlevents.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.sandl.snlevents.model.db.UserTransaction;
+import uk.gov.hmcts.reform.sandl.snlevents.model.request.HearingSessionRelationship;
+import uk.gov.hmcts.reform.sandl.snlevents.model.response.HearingInfo;
+import uk.gov.hmcts.reform.sandl.snlevents.repository.db.HearingRepository;
+import uk.gov.hmcts.reform.sandl.snlevents.service.HearingPartService;
+
+import java.util.UUID;
+
+import static org.springframework.http.ResponseEntity.ok;
+
+@RestController
+@RequestMapping("hearing")
+public class HearingController {
+
+    @Autowired
+    private HearingRepository hearingRepository;
+
+    @Autowired
+    private HearingPartService hearingPartService;
+
+    @GetMapping(path = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public HearingInfo getHearingById(@PathVariable("id") UUID id) {
+        return new HearingInfo(hearingRepository.findOne(id));
```

Comment:
can we move it to service?

Reply:
There's no HearingService right now and as it's just one method no need to add it.